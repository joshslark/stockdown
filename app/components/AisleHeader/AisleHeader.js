import React, {Component} from 'react';

import {
  Text,
  View,
} from 'react-native';
import styles from './styles';

export default class AisleHeader extends Component {
  render() {
    return (
      <View
        style={ styles.listHeaderBox }>
      <Text
        style={ styles.listHeaderText }>
        Aisle {this.props.number}
      </Text>
      </View>
    );
  }
}

