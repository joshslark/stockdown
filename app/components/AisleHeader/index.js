export {default as AisleHeader} from './AisleHeader';
