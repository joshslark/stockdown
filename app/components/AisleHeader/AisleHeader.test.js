import React from 'react';
import AisleHeader from './AisleHeader';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<AisleHeader number="00" />).toJSON();
  expect(tree).toMatchSnapshot();
});
