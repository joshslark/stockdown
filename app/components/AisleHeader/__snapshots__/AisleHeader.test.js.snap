// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgb(233,207,178)",
      "borderBottomWidth": 0.5,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "center",
        "borderBottomWidth": 1,
        "color": "black",
        "fontSize": 36,
        "padding": 10,
      }
    }
  >
    Aisle 
    00
  </Text>
</View>
`;
